pints/_optimisers/__init__.py

```diff
@@ -279,6 +279,29 @@ def set_hyper_parameters(self, x):
         self.set_population_size(x[0])
 
 
+class LineSearchBasedOptimiser(Optimiser):
```

Comment:
I'm not 100% sure this is the best way to do it yet, but let's leave all that till after we have a working prototype!